specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -4800,10 +4803,22 @@
           "multipleOf": 1.0,
           "minimum": 0,
           "description": "Value indicating the age in days after blob creation."
+        },
+        "autoUptier": {
+          "$ref": "#/definitions/AutoUptier",
+          "description": "Automatically up tiers the blob to the account level access tier after the blob is read. This is a generalized implemntation of enableAutoTierToHotFromCool and is mutually exclusive with enableAutoTierToHotFromCool."
```

Comment:
Please also mention in description:
1. AutoUptier can only works with daysAfterLastAccessTimeGreaterThan (From example it looks so. If not, please include any limitation of the usage to the description)
2. Which action can use AutoUptier :tierToCool, tierToArchive, tierToHot, tierToCold?

Reply:
added to description